openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCardNew/ActivityFeedcardNew.component.tsx

```diff
@@ -307,12 +311,16 @@ const ActivityFeedCardNew = ({
           ) : (
             <div className="d-flex gap-2">
               <div>
-                <ProfilePictureNew
-                  avatarType="outlined"
-                  key={feed.id}
-                  name={getEntityName(currentUser)}
-                  size={32}
-                />
+                <UserPopOverCard userName={getEntityName(currentUser)}>
+                  <div className="d-flex items-center">
+                    <ProfilePicture
+                      avatarType="outlined"
```

Comment:
same

Reply:
Had missed to remove it, removed it now.